src/Polly.Core/Polly.Core.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net7.0</TargetFrameworks>
+    <UsePollyTarget>true</UsePollyTarget>
+    <AssemblyTitle>Polly.Core</AssemblyTitle>
+  </PropertyGroup>
+
+  
+  <ItemGroup Label="PackageReferences">
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="7.0.0" />
+  </ItemGroup>
```

Comment:
Same question here regarding TFM and multi-targeting.

Reply:
The `Microsoft.Extensions.Logging.Abstractions` is not implicit in any framework and needs to be explicitly added.